external-crates/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -209,13 +374,13 @@ impl Interpreter {
 
         let link_context = data_store.link_context();
         let mut current_frame = self
-            .make_new_frame(link_context, loader, function, ty_args, locals)
+            .make_new_frame(function, ty_args.clone(), locals)
```

Comment:
why this `clone`?

Reply:
Great catch, turns out I don't need to clone ty_args